scss/main.scss

```diff
@@ -93,14 +93,11 @@ a {
 }
 
 #main {
-  padding-bottom: 160px;
+  padding-top: 30px;
+  padding-bottom: 420px;
```

Comment:
We want the footer to always stay at the page bottom. Without reserving additional space in `#main` will result in something like

<img width="355" alt="screen shot 2017-02-22 at 3 32 07 pm" src="https://cloud.githubusercontent.com/assets/2896608/23237909/5e23b92e-f914-11e6-895a-0f5fdf49eb6d.png">

vs with reserved space

<img width="362" alt="screen shot 2017-02-22 at 3 32 26 pm" src="https://cloud.githubusercontent.com/assets/2896608/23237916/68c64ebe-f914-11e6-84bd-835a2e564775.png">

I can't think of a better solution without having annoying hardcoded reserved space to achieve this. If we don't have absolute poisoning on `footer`, `footer` can end up being in a middle of the page on large devices. Plus we are loading dynamic content on most of the pages - that adds in some complexity too. Suggestion?